tests/test_local_folder.py

```diff
@@ -81,6 +82,23 @@ def test_local_download_paths_are_cached(tmp_path: Path):
     assert paths1 is paths2
 
 
+def test_local_download_paths_long_paths(tmp_path: Path):
+    """Test long path handling on Windows."""
+    if os.name == "nt":
```

Comment:
```suggestion
@pytest.mark.skipif(os.name != "nt", reason="Windows-specific test.")
def test_local_download_paths_long_paths(tmp_path: Path):
    """Test long path handling on Windows."""
```

You can skip the test in a proper way using `pytest.mark.skipif`. This way the test is labelled as "skipped" (or "success") depending on the machine it runs on.